mesh/mesh.cpp

```diff
@@ -7265,22 +7265,23 @@ void Mesh::PrintVTK(std::ostream &out)
       for (int i = 0; i < NumOfElements; i++)
       {
          Nodes->FESpace()->GetElementDofs(i, dofs);
+         MFEM_ASSERT(Dim != 0 || dofs.Size() == 1,
+                     "Point meshes should have a single dof per element");
          size += dofs.Size() + 1;
       }
       out << "CELLS " << NumOfElements << ' ' << size << '\n';
       const char *fec_name = Nodes->FESpace()->FEColl()->Name();
 
-      if (!strcmp(fec_name, "H1_0D_P1"))
-      {
-         order = 0;
-      }
-      else if (!strcmp(fec_name, "Linear") ||
-               !strcmp(fec_name, "H1_2D_P1") ||
-               !strcmp(fec_name, "H1_3D_P1"))
+      if (!strcmp(fec_name, "Linear") ||
+          !strcmp(fec_name, "H1_0D_P1") ||
```

Comment:
Ok -- so ``H1_0D_P1`` is order 1, rather than order 0.

I suppose they are essentially equivalent for a point which only has a single DOF.

Reply:
Using order=1 makes sense because it simplifies the implementation. It also makes sense because the lowest order of H1 spaces is 1 for all other dimensions.